client/state/data-layer/wpcom/comments/test/index.js

```diff
@@ -97,6 +99,74 @@ describe( 'wpcom-api', () => {
 			} );
 		} );
 
+		describe( '#writePostComment()', () => {
+			const action = {
+				type: 'DUMMY',
+				siteId: 101010,
```

Comment:
Does it really matters? Since we're never actually dispatching the action. If it does matter, I'd rather change them with something nonsensical, like a large number. Or a string (which will break if we do type checks in the future)